src/sentry/profiles/task.py

```diff
@@ -349,6 +417,7 @@ def run_symbolicate(
     profile: Profile,
     modules: List[Any],
     stacktraces: List[Any],
+    platform: Optional[str] = None,
```

Comment:
Here as well as the caller should have ensured the platform is defined already
```suggestion
    platform: str,
```